src/modules/API.js

```diff
@@ -0,0 +1,8 @@
+const baseUrl = 'https://www.eventbriteapi.com/v3/users/me/?token=PWQEI5BDOZEEQ2A5NVQT';
```

Comment:
You can define the token inside a .env file and add the .env file to your gitignore.